server/data/restClient.ts

```diff
@@ -1,4 +1,4 @@
-import superagent from 'superagent'
+import superagent, { Response, ResponseError } from 'superagent'
```

Comment:
A lot of refactoring in this class to be able to support `PUT` and `DELETE` .... but its mostly a copy and paste job from Send Legal Mail! 😁 👍